src/Modal.jsx

```diff
@@ -1,12 +1,48 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
+import { Document, Page, pdfjs } from "react-pdf";
+import { API_BASE_URL } from "./constants";
+import "react-pdf/dist/esm/Page/AnnotationLayer.css";
+import "react-pdf/dist/esm/Page/TextLayer.css";
+
+pdfjs.GlobalWorkerOptions.workerSrc = `//cdnjs.cloudflare.com/ajax/libs/pdf.js/${pdfjs.version}/pdf.worker.js`;
 
 const Modal = ({
   selectedInventionTitle,
   selectedInventionAbstract,
   selectedOpsLink,
+  selectedImageLinks,
   handleClose,
 }) => {
   const [showModal, setShowModal] = useState(true);
+  const [imageData, setImageData] = useState(null);
+
+  useEffect(() => {
+    const USER_ID = sessionStorage.getItem("uid");
+
+    if (selectedImageLinks && selectedImageLinks.length > 0) {
+      fetch(
+        `${API_BASE_URL}/firstpageClipping?uid=${USER_ID}&fpcImage=${selectedImageLinks[0].link}&fpcImageFormat=${selectedImageLinks[0].format}`,
```

Comment:
Here you're using the first item of the array. Instead you should use the item whose "desc"="FirstPageClipping" with the relative format (usually png).
Check my reply example:
{
    "images_links": [
        {
            "desc": "FullDocument",
            "format": "pdf",
            "link": "published-data/images/US/2023155060/A1/fullimage",
            "nofpages": "40"
        },
        {
            "desc": "Drawing",
            "format": "pdf",
            "link": "published-data/images/US/2023155060/A1/thumbnail",
            "nofpages": "22"
        },
        {
            "desc": "FirstPageClipping",
            "format": "png",
            "link": "published-data/images/US/2023155060/PA/firstpage",
            "nofpages": "1"
        }

You may leave the item[0] as an alternative, if "FirstPageClipping" does not exist.

Of course this change requires that you adapt the code below to display other format than pdf